### PR TITLE
11.0 website sale product tags port

### PR DESCRIPTION
**Pull request by KolushovAlexandr**

### CODE REVIEW

**Comment by yelizariev** [comment]
?

**Comment by yelizariev** [comment]
odoo.exceptions.UserError: ("You try to install module 'website_sale_product_tags' that depends on module 'product_template_tags'.\nBut the latter module is not available in your system.", '')

**Comment by KolushovAlexandr** [comment]
@yelizariev it awaits this PR https://github.com/OCA/product-attribute/pull/305

**Comment by yelizariev** [comment]
According to @KolushovAlexandr, we don't wait mergign PR in OCA anymore

File: website_sale_product_tags/doc/changelog.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+`1.0.0`
+-------
+
+- Init version
```
[the remark this answers is not in the document]
Comment (Ramil-Mukhametzyanov) [reply website_sale_product_tags/doc/changelog.rst]: :+1: checked

**Comment by ilmir-k** [comment]
I approve to merge it now

**Pull request merged**